src/pages/Home.js

```diff
@@ -0,0 +1,131 @@
+import { Link } from 'react-router-dom'
+import {Navigation} from '../components/Navigation'
+import '../components/clhome.css'
+import '../components/navigation.css'
+import biryaniimage from '../assets/images/biryaniimg.jpeg'
+import thaaliimage from '../assets/images/thalis.jpeg'
+import cakesimage from '../assets/images/cakes.jpeg'
+import chineseimage from '../assets/images/chinese.webp'
+import imageavtaar from '../assets/images/avatar-the-grinch-green-plant-recycling-symbol-food-transparent-png-1652162.png'
+import React,{useState} from "react"
+
+export default function Home(){
+
+    const [displayLogIn,setDisplayLogIn] = useState("none")
+    const [displaySignIn,setDisplaySignIn] = useState("none")
+    function clickLogInHandler(){
+        if(displayLogIn === "none")
+        setDisplayLogIn("block")
+        if(displayLogIn==="block")
+        setDisplayLogIn("none")
+    }
+    function clickSignInHandler(){
+        if(displaySignIn==="none")
+        setDisplaySignIn("block")
+        if(displaySignIn==="block")
+        setDisplaySignIn("none")
+    }
+    return(
+        <div className="container">
+            <div className="cards">
+
+<div className="card">
+
+    <img src={biryaniimage} alt="biryani" style={{width:"100%"}}/>
+    <h1>Biryanis</h1>
+    <p className="price">₹100* Onwards</p>
+    <p>Order from best resturants</p>
+    <p><Link to="/ProductListing">more</Link></p>
+  </div>
+  <div className="card">
+                <img src={thaaliimage} alt="biryani" style={{width:"100%"}} />
```

Comment:
try to not use inline styles, it is considered bad practice mostly due to specificity reasons